### ISSUE TITLE
Documentation Issue for New Stats

### ISSUE THREAD

**Issue by awohns** [tskit-dev/tskit#222]
The documentation of the f2 function in python/trees.py looks like its just a copy of the f3 one: 

```
def f2(self, sample_sets, indexes=None, windows=None, mode="site",
           span_normalise=True):
        """
        Computes Patterson's f3 statistic between four groups of sample_sets.
        See :ref:`sec_general_stats` for details of ``indexes``, ``windows``,
        and ``mode`` and return value. Operates on ``k = 3`` sample sets
        at a time.
        What is computed depends on ``mode``. Each works exactly as
        :ref:``f4``, except the average is across randomly chosen set of four
        samples ``(a1, b1; a2, b2)``, with `a1` and `a2` both chosen (without
        replacement) from the first sample set and ``b1`` and ``b2`` chosen
        randomly without replacement from the second samle set. See :ref:``f4``
        for more details.
```

**Comment by petrelharp** [comment]
Thanks for the scrutiny... but, is it? It'd got the `k=3` in there, which should clearly be `k=2`, but after that I think it is different. Do you agree it is correct?

(hm, I alse see a "samle set" in there, whoops...)

**Comment by awohns** [comment]
Well, doesn't f_2 operate on two sample sets, not four as is stated in the documentation? Apologies if I'm off here.
Otherwise, I think the k = 2 is the only change (and "samle").

**Comment by petrelharp** [comment]
Ah, yep - hadn't properly parsed that first sentence (put blocks of code between rows of at least three backticks). If we change the first bit to
```
        Computes Patterson's f2 statistic between two groups of sample_sets.
        See :ref:`sec_general_stats` for details of ``indexes``, ``windows``,
        and ``mode`` and return value. Operates on ``k = 2`` sample sets
        at a time.
```
do you agree with the rest?

Again, thanks!

**Comment by petrelharp** [comment]
I've fixed this over at  7996e2e

**Comment by petrelharp** [comment]
fixed in cd3023920f0237f1599e15309c3f9cc95896a767
